domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/party/Claimant.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.cmc.domain.models.party;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import uk.gov.hmcts.cmc.domain.models.Address;
+import uk.gov.hmcts.cmc.domain.models.legalrep.Representative;
+
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Claimant extends Party {
```

Comment:
If party itself is enough without identifying whether it is individual, organisation or others subtypes we should make Party non abstract and allow instantiating it. Then if someone has to use more specialised type they can select one of 4 subtypes.

Other thing is the name, having claimant subtype deriving from party where other subtypes are split using different perspective does not make sense. If the first approach is not desirable by the team then we should at least call this class `UnspecifiedParty` or similar.